osu.Game.Tests/Visual/Multiplayer/QueueModeTestScene.cs

```diff
@@ -86,7 +85,6 @@ public override void SetUpSteps()
             ClickButtonWhenEnabled<MultiplayerMatchSettingsOverlay.CreateOrUpdateButton>();
 
             AddUntilStep("wait for join", () => MultiplayerClient.RoomJoined);
-            AddUntilStep("wait for ongoing operation to complete", () => !(CurrentScreen as OnlinePlayScreen).ChildrenOfType<OngoingOperationTracker>().Single().InProgress.Value);
```

Comment:
Why is this in the diff? It looks like this pull is reverting https://github.com/ppy/osu/pull/22450 for whatever reason?

Please fix the history here. Probably easiest to rebase your work onto current `master` and force-push to fix (this is a special exemption, do *not* do this normally).

Reply:
Sorry about that. It seems that when trying the rebase my changes onto the latest master I had gotten some of the newer changes caught up in my commit. I believe I have now fixed the issue.